### DISCUSSION TITLE
Welcome to lppls Discussions!

### DISCUSSION THREAD

**Discussion by Joshwani** [Boulder-Investment-Technologies/lppls#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hideyukitak** [comment]
@Joshwani Hi Josh! Would love to chat with you about the model you coded. I'm a high schooler that just started my journey in coding so I would love to get some advice! Apparently, GitHub doesn't have a message feature and I couldn't find your email so feel free to contact me at: darthvaderfishmarket@gmail.com

Excuse my email address.

Thanks!

**Comment by Zhoux26** [comment]
Hi @Joshwani, 
It's pretty a good package of LPPL, awesome!

However, I've got two issues here and would like to ask.

1. I tried to follow the instruction to do LPPL fit, it's good when using nasdaq daily quote, the same result as yours.
   Then I was trying to use my own data and the plot was like quite smooth curve not a very exponential function in your example.
   The data I've used contains more than 250 points and the trend is also like nasdaq. So is there something I've missed in data selection.
   Like data range, parameter selection? 
   
  
2. Also, I read many papers, they said using GA to estimate the parameters. I'am not quite sure if the package has this selection or We 
    used what method?

Hope for your reply,
Thanks so much!
